### ISSUE TITLE
Add a way to view history that is fully contained in a bounding box

### ISSUE THREAD

**Issue by jrmuizel** [openstreetmap/openstreetmap-website#837]
Currently the history view usually contains a bunch of changes that are uninteresting to the user. They show up because they happen to intersect the bounding box the user is looking at. It would be very helpful to at least have an option of only seeing changes that are fully contained in the bounding box so that global changes are skipped.

**Comment by jojo4u** [comment]
Just use Whodidit, the website "history view" is quite useless in comparison and could be removed imho.

**Comment by tbertels** [comment]
Indeed, wouldn't it be possible to adapt [WhoDidIt](https://wiki.openstreetmap.org/wiki/Quality_assurance#WhoDidIt) to the History tab, with more up to date data? At least until this issue is fixed. Or maybe just a link redirecting to the current bbox?
Even with this issue fixed, the History tab usefulness is very limited: probably as a side effect of this issue, it hasn't been improved for quite some time.
